public/docs/ts/latest/quickstart.jade

```diff
@@ -235,7 +236,7 @@ code-example(format="").
 :marked
   There are three noteworthy sections of HTML:
 
-  1. We load the JavaScript libraries we need.<br/>
+  1. We load the JavaScript libraries we need. `angular2-polyfills.js` and `Rx.js` are needed by Angular 2.<br/>
```

Comment:
We don't. Yet. If ever. If we do, I don't think we'll  do it in the quickstart. Not today anyway.